dbt_airflow_factory/k8s/k8s_parameters.py

```diff
@@ -62,7 +65,7 @@ def __init__(
         envs: Optional[Dict[str, str]] = None,
         secrets: Optional[List[Secret]] = None,
         is_delete_operator_pod: bool = True,
-        config_file: str = "~/.kube/config",
+        config_file: Optional[str] = None,
```

Comment:
this looks good but I recall that when we set it to none it didn't work on gcp/composer. Maybe there was a different problem then but it needs to be verified after change/release.

Reply:
Cloud Composer needed:
```yaml
namespace: default
config_file: '/home/airflow/composer_kube_config'
```

MWAA needed:

Prerequisites `kube_config.yaml` pushed to s3 bucket
```yaml
namespace: mwaa
config_file: "/usr/local/airflow/dags/kube_config.yaml"
in_cluster: False
cluster_context: mwaa
```